### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: src/components/AppHeader/UserMenu.js [hunk 1]
```diff
@@ -8,15 +8,20 @@ import { ACC_API } from '../../constants/config';
...
 const UserMenu = ({ user }) => {
 	const menu = (
 		<Menu>
 			<Menu.Item>
 				<Link to="/profile">{user.email}</Link>
 			</Menu.Item>
 			<Menu.Divider />
-			<Menu.Item>
-				<a href={logoutURL}>Logout</a>
+			<Menu.Item onClick={() => handleLogout()}>
```
Comment (metagrover) [inline src/components/AppHeader/UserMenu.js]: It should be: `onClick={handleLogout}`

File: src/pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -185,9 +193,9 @@ class HomePage extends Component {
 									margin: 0,
 								}}
 							>
-								All Apps
+								{sortedApps.length ? 'All Apps' : 'Create an app to get Started'}
```
Comment (metagrover) [inline src/pages/HomePage/HomePage.js]: I don't think this is the best way to represent "Create a new app to get started". We need something similar to the `NoMatch` page here.

File: src/pages/OnboardingPage/index.js [hunk 3]
```diff
@@ -30,8 +30,7 @@ export default class Onboarding extends Component {
 
 	nextScreen = () => {
 		this.setState((state) => {
-			const currentScreen =
-				state.currentScreen + 1 < state.totalScreen
+			const currentScreen =				state.currentScreen + 1 < state.totalScreen
```
Comment (metagrover) [inline src/pages/OnboardingPage/index.js]: Linting

File: src/pages/OnboardingPage/index.js [hunk 4]
```diff
@@ -206,12 +211,12 @@ export default class Onboarding extends Component {
 					</ul>
 				</div>
 				<div className="right">
-					<a className="skip-link" href="/">
+					<button type="button" className="skip-link" onClick={this.skipTutorial}>
```
Comment (metagrover) [inline src/pages/OnboardingPage/index.js]: You don't need `type="button"` with a `button`.

Comment (metagrover) [reply src/pages/OnboardingPage/index.js]: Also, this has affected the design:

image

Comment (jyash97) [reply src/pages/OnboardingPage/index.js]: If we remove `type="button"` it gives linting error

**Pull request merged**